### PR TITLE
Add e2e-metal-ipi-serial-ipv4 into release

### PR DESCRIPTION
**Pull request by ardaguclu**
This PR adds e2e-metal-ipi-serial-ipv4 job as an informer.

### CODE REVIEW

File: core-services/release-controller/_releases/release-ocp-4.8.json [hunk 1]
```diff
@@ -68,6 +68,10 @@
       "maxRetries": 3,
       "prowJob":{"name":"periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi"}
     },
+    "metal-ipi-serial-ipv4": {
```
Comment (andfasano) [inline core-services/release-controller/_releases/release-ocp-4.8.json:71]: For consistency with the previous job I guess a better naming would be `metal-ipi-serial`

**Review by petr-muller — APPROVED** [review]

**Pull request merged**